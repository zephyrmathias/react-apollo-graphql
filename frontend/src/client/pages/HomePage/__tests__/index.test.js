import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../index';

describe('pages: <HomePage />', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
