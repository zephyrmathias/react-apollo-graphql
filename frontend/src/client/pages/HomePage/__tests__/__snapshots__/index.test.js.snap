// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages: <HomePage /> should render correctly 1`] = `
<div>
  HomePage
</div>
`;
